From 9b63c35dc800ee7ab0d5075809d30e23b101c758 Mon Sep 17 00:00:00 2001
From: Josh Fazekas <Josh.Fazekas@Intel.com>
Date: Fri, 23 May 2025 02:43:15 +0300
Subject: [PATCH] [SW-184431] g3: allocate real size for rma table

Change memory allocation for rma table to use actual rma table size
instead of fixed 25KB size

Change-Id: I6adc1564a455b9ca8553c0d97b1ba9cf5d8639a0
Signed-off-by: Josh Fazekas <Josh.Fazekas@Intel.com>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 0ce337c..c14baea 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -114,8 +114,7 @@
 
 static struct rst_src_data rst_src;
 
-#define RMA_TABLE_SIZE	(25 * 1024)
-static uint8_t rma_table[RMA_TABLE_SIZE];
+static uint8_t rma_table[RMA_MAX_TOTAL_SZ];
 
 struct sys_heap mem_heap_sec;
 
